oat-sa/extension-tao-itemqti
# Fix/ms 707/multi paragraphs in extended text

# Pull request summary

Related to [MS-707](https://oat-sa.atlassian.net/browse/MS-707).

When TT's response contains text with multiple paragraphs it will be displayed without line breaks

# How to test

1. Create test with extended text interaction
2. Pass the test as a TT (fill in the field with multiple lines)
3. Log in as a Scorer
4. Start scoring corresponding task
5. Check that line breaks and spaces are displayed correctly

# Checklist

- [ ] Pull request's target is not master
